Lib/concurrent/interpreters/_crossinterp.py

```diff
@@ -40,8 +40,10 @@ class UnboundItem:
 
     @classonly
     def singleton(cls, kind, module, name='UNBOUND'):
-        doc = cls.__doc__.replace('cross-interpreter container', kind)
-        doc = doc.replace('cross-interpreter', kind)
+        doc = cls.__doc__
```

Comment:
we pass `doc` as the value for `__doc__` on line 53 so I think it's important to keep the possibility of `None` existing